app/services/MatcherPool.scala

```diff
@@ -106,6 +107,8 @@ class MatcherPool(
 
     val eventuallyResponses = jobs.map(offerJobToQueue)
 
+    val totalCharsForCheck = query.blocks.foldLeft(0)((acc, block) => acc + block.text.size)
+
```

Comment:
I assume the changes in the file are for some future PR?

Reply:
Oops, good spot – yes, they're for our slow log, which will come later.